docs/guides/run_a_job.md

```diff
@@ -0,0 +1,227 @@
+!!! important "Quick Look"
+    `floyd run [OPTIONS] [COMMAND]`
+
+    **[[OPTIONS]](#options_1):**
+
+    - [Instance Type](#instance-type): `--cpu --gpu --cpu+ --gpu+`
+    - [Dataset(s)](#datasets): `--data <name_of_datasource>:<mount_point_on_server>`
+    - [Mode](#mode): `--mode <mode_name>`
+    - [Environment](#environment): `--env <environment_name`
+    - [Message](#message) `--message` or `-m`
+    - [Tensorboard](#tensorboard): `--tensorboard`
+
+    **[[COMMAND]](#command_1)**
+
```

Comment:
Would it make sense to add a couple of examples here? May be add another "box" called examples (similar to the "Quick look", but a different color) E.g. To run a Jupyter Notebook with Pytorch `floyd run --env pytorch-0.2 --mode jupyter`, etc.